.github/workflows/tox-tests.yml

```diff
@@ -101,3 +102,62 @@ jobs:
       with:
         name: logs.zip
         path: .tox/ansible${{ matrix.ansible-version }}/log/
+
+  publish-dists:
+    name: Publish 🐍📦 to (Test)PyPI
```

Comment:
Please avoid emoji an use plain english, please.

Reply:
No, I want the text to fit in GH UI.

Reply:
Well `Publish to (test.)pypi.org" does fit an is shorter, no need to mention what. Just removing tee unicode-emoji part would keep the text clear.

Reply:
```suggestion
    name: Publish to (Test)PyPI
```

Reply:
I guess so. Also, not PyPi but PyPI.
My point is that this is out of scope anyway because it follows the existing style. It's okay to debate style change in a separate PR and @ssbarnea already has one for that. I don't want to mix different kinds of changes in the middle of random unrelated PRs.
I currently follow the style that exists here and in other projects where I maintain GHA. Blocking a PR just because it follows the rest of the style is nonsense.

Reply:
I don't like doing this, though I'm just going to pull rank here.
Let's get the functional changes in so we can drop Travis. Then we can address this in a follow up PR.

@webknjaz thanks for the clarification that it's `PyPI`, I wasnt aware of that.